features/step_definitions/confirm_dialog_steps.rb

```diff
@@ -8,3 +8,14 @@
     step original
   end
 end
+
+Then /^(.+) and I reject dialog box(?: "(.*)")?$/ do |original, text|
```

Comment:
I see we already do the same for accepting the dialog. So it's fine to keep as is.

Just an idea, how about making step:
```ruby
Given "/^I reject dialog box(?: "(.*)")?$/ do |text, steps_string|
  steps steps_string
end
```

Then call it as:
```ruby
When I reject dialog box "This action deletes the corresponding routes in OpenShift and reverts the Public Base URLs back to the latest value. Do you want to continue?"
"""
  When I click on the label "APIcast self-managed" 
"""
```

Where in the multi-line string you can have multiple steps.

Reply:
By now I'd keep it like it is. There are more than 50 tests using the "accept confirm" way, but we can improve it in a following task.